odoo/addons/test_http/tests/test_webjson2.py

```diff
@@ -201,6 +203,24 @@ def test_webjson2_good(self):
         self.assertEqual(res.status_code, HTTPStatus.OK)
         self.assertEqual(res.headers.get('Content-Type'), 'application/json; charset=utf-8')
 
+        res = self.db_url_open(
+            '/json/2/res.users/read',
+            data=r'{"ids": [%d], "fields": ["display_name"]}' % self.jackoneill.id,
+            headers=CT_JSON | self.bearer_header,
+        )
```

Comment:
res.raise_for_status()
69972274772a600757f8b2a2c41e4aa59b53a900 made it better
if you can also change the first `res = ` in `test_websjon2_good` I would be grateful :pray: